charts/spire/charts/spire-server/values.yaml

```diff
@@ -498,13 +505,20 @@ ingress:
     # nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
     # nginx.ingress.kubernetes.io/ssl-passthrough: "true"
 
-  ## @param ingress.hosts [array] Host paths for ingress object
-  hosts:
-    - host: spire-server.example.org
-      paths:
-        - path: /
-          pathType: Prefix
-  ## @param ingress.tls [array] Secrets containining TLS certs to enable https on ingress
+  ## @param ingress.host Host to use for autogenerated rules. If no '.' is in host, trustDomain is automatically appended.
+  host: "spire-server"
+
+  ## @param ingress.tlsSecret secret to use for autogenerated rules.
+  tlsSecret: ""
+
+  ## @param ingress.hosts [array] Host paths for ingress object. If emtpy, rules will be built based on the host var.
```

Comment:
```suggestion
  ## @param ingress.hosts [array] Host paths for ingress object. If emtpy, rules will be built based on the host var. Only one of host or hosts can be specified.
```

Reply:
Suggest to move this up under host

Reply:
The whole value or just the comment?

Reply:
actually just leave it, I think it works better as the host/tlsSecret and hosts/ tls pairs